de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/nodes/environment/ExistingEnvironmentManager.java

```diff
@@ -62,4 +63,16 @@ public Optional<Path> getEnvironment() {
   public void deleteEnvironment(Path path) {
     // The path existed previously and should not be removed.
   }
+
+  @Override
+  public String[] getEntries() {

```

Comment:
@ahmadswaid: The Mapper complains about the missing property "entries": 
![image](https://user-images.githubusercontent.com/38949263/197730226-faca4b19-cd9e-4e9e-80b7-034966a99fe7.png)